.github/workflows/note-tutorials-python-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: note-tutorials-python-ci
+
+on: [push]
+
+jobs:
+  python-ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"] # ["3.7", "3.8", "3.9", "3.10"]
+        sample:
+          - "sensor-tutorials/python/i2c-with-library"
+          - "sensor-tutorials/python/no-library"
+          - "sensor-tutorials/python/serial-with-library"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          cd ${{ matrix.sample }}
+          if [ -f requirements-ci.txt ]; then pip install -r requirements-ci.txt; fi
+      - name: Lint with flake8
+        run: |
+          cd ${{ matrix.sample }}
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      # - name: Test with pytest
+      #   run: |
+      #     cd ${{ matrix.sample }}
+      #     pytest
```

Comment:
Was this intentionally left as commented out?

Reply:
Nope! That'll teach me not to skip the self review

Reply:
Oh I misread and was thinking about another CI - yes there are presently no tests, and including this fails the build, since pytest errors when there are no tests.